DashDocs/src/utils/parse_example.jl

```diff
@@ -0,0 +1,131 @@
+
+const top_level_exprs = Set([:using, :import, :const])
+# pull up and return top level calls from Expr
+function top_level_popout!(e::Expr)
+    top_inds = findall(e.args) do arg
+        return isa(arg, Expr) && arg.head in top_level_exprs
+    end
+    result = e.args[top_inds]
+    deleteat!(e.args, top_inds)
+    return result
+end
+
+function check_is_download(right::Expr)
+    (right.head != :call || right.args[1] != :DataFrame || !isa(right.args[2], Expr)) && return nothing
```

Comment:
Please add comments here. What if the user constructs a DataFrame manually?

Reply:
done in c41c702